lib/src/main/java/graphql/nadel/validation/util/NadelGetReachableTypes.kt

```diff
@@ -223,6 +226,15 @@ internal fun getReachableTypeNames(
             add(node.name)
             return CONTINUE
         }
+
+        override fun visitGraphQLAppliedDirective(
+            node: GraphQLAppliedDirective,
+            context: TraverserContext<GraphQLSchemaElement>,
+        ): TraversalControl {
+            // Don't look into applied directives. Could be a shared directive.
+            // As long as the schema compiled then we don't care.
+            return ABORT
+        }
```

Comment:
We need this otherwise it'll start looking at directive argument types etc.

We don't want to validate those as we have tons of shared directives.

As long as the `GraphQLSchema`  compiles then we are happy.